### DISCUSSION TITLE
批量插入数据到milvus 报grpc的错误

### DISCUSSION THREAD

**Discussion by javayydd** [milvus-io/milvus#26511] [Q&A and General discussion] [answered]
grpc_message:"grpc: received message larger than max、
![image](https://github.com/milvus-io/milvus/assets/87173166/1ac5a044-ef0b-40a9-b5cd-18a7c697db65)
请问是贵项目限制的大小吗

**Comment by xiaofan-luan** [comment, marked as answer]
grpc限制，单词插入不超过64m。
个人建议不建议超过4M
